vcs/src/main/java/org/openjdk/skara/vcs/openjdk/OpenJDKTag.java

```diff
@@ -52,16 +52,19 @@ private OpenJDKTag(Tag tag, String prefix, String version, String buildPrefix, S
      * jdk7u40-b20 -> jdk7u40  7u40     u20     -b           29
      * hs24-b30    -> hs24     24               -b           30
      * hs23.6-b19  -> hs23.6   23.6     .6      -b           19
+     * 11.1+22     -> 11.1     11.1     .1      +            22
      */
 
     private final static String legacyOpenJDKVersionPattern = "(jdk([0-9]{1,2}(u[0-9]{1,3})?))";
     private final static String legacyHSVersionPattern = "((hs[0-9]{1,2}(\\.[0-9]{1,3})?))";
     private final static String legacyBuildPattern = "(-b)([0-9]{2,3})";
     private final static String OpenJDKVersionPattern = "(jdk-([0-9]+(\\.[0-9]){0,3}))(\\+)([0-9]+)";
+    private final static String OpenJFXVersionPattern = "((?:jdk-){0,1}([1-9](?:(?:[0-9]*)(\\.(?:0|[1-9][0-9]*)){0,3})))(?:(\\+)([0-9]+)|(-ga))";
```

Comment:
Not related to this change, but the OpenJDKVersionPattern also will need support for `*-ga` tags.